README.md

```diff
@@ -1,76 +1,101 @@
-# HTML5 Media
+# Working with HTML5 Media
+
+## Problem Statement
+The internet is a highly interactive environment. Even blogs where text
+may be the most dominant content, audio and video elements often support,
+or even supplement, the content. In the case of 3rd party media sites such
+as YouTube, SoundCloud, and Vimeo, you are given an all-in-one line of code
+with configurable properties that creates the embeddable content for you--
+but how do you do this manually in your application? Often times, you will be working
+with, and manipulating, media that is being uploaded into or generated by your
+app. HTML has evolved dramatically since the days of the generic `embed` tag.
+How is this done today?
 
 ## Overview
+1. Demonstrate how to embed audio elements in HTML5
+2. Demonstrate how to embed video elements in HTML5
+3. Link to audio and video converters
+
+## Demonstrate How to Embed Audio Elements in HTML5
+Embedding audio can be a nice touch to a website. If you had a MySpace page back
+in the 2000s, you can probably to this. HTML was very different back
```

Comment:
Let's not date the audience or imply that coming now is late.

Also, sentence is not grammatical.